runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsConfig.java

```diff
@@ -60,4 +63,26 @@ default boolean enableRestorationBars()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		position = 4,
+		keyName = "toggleRestorationBars",
+		name = "Toggle to hide when not in combat",
+		description = "Visually hides the Status Bars when player is out of combat."
+	)
+	default boolean toggleRestorationBars()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 5,
+		keyName = "hideStatusBarDelay",
```

Comment:
This is not a descriptive key name. It should reference that this is specifically the combat hiding delay.